src/workflow/auth.js

```diff
@@ -49,19 +49,39 @@ const questions = [
   }
 ]
 
+function runAuthenticate (answers) {
+  spinner.start()
+  const authRequest = client.authenticate(answers)
+  authRequest.then(spinner.stop, spinner.stop)
+  return authRequest
+}
+
 function auth () {
+  if (
+    process.env.MASHERY_USERNAME != undefined &&
```

Comment:
Would it be valid if some of these variables are set to empty string?
Otherwise, we could use `!process.env.<variable>` instead